.env

```diff
@@ -1,2 +1,5 @@
 VUE_APP_I18N_LOCALE=ru
 VUE_APP_I18N_FALLBACK_LOCALE=en
+VUE_APP_GRANT_TYPE=password
+VUE_APP_CLIENT_ID=3
+VUE_APP_CLIENT_SECRET=W90b7o7e81OI8JwNXItC8GXouS8rUStS9kQPKFul
```

Comment:
Hey, that is a public repo, please remove it from here we will use github secrets for CI

Reply:
done